test/chrono-test.cc

```diff
@@ -641,3 +641,12 @@ TEST(chrono_test, cpp20_duration_subsecond_support) {
 }
 
 #endif  // FMT_STATIC_THOUSANDS_SEPARATOR
+
+#if (FMT_USE_UTC_TIME)
```

Comment:
Let's remove the redundant parentheses around FMT_USE_UTC_TIME for consistency with the rest of the code.